### ISSUE TITLE
Victoria 3 Strange Localisation parsing errors

### ISSUE THREAD

**Issue by kaiser-chris** [DragonKnightOfBreeze/Paradox-Language-Support#98]
Hi,

I noticed a few places where localisation fields (as defined in CWT) cannot be resolved even though they exist.
Below is one example but I noticed a few. If you need the corresponding files or more examples I can search for them and post them as well.

### Test:

- Copied 00_devout.txt into my mod
  - Other localization keys can be resolved but this one can not be resolved
- Copied interfaces_l_english.yml also into my mod
  - Then the localization key can be resolved

### game/common/interest_groups/00_devout.txt (Base Game File)
![image](https://github.com/user-attachments/assets/b6fe8ad2-0381-46a7-b3b8-7ffc7d80b303)
### game/localization/english/interfaces_l_english.yml (Base Game File)
![image](https://github.com/user-attachments/assets/aa1961c4-6b16-425e-80f5-69237692e9a9)

**Comment by kaiser-chris** [comment]
I have noticed this behaviour with multiple type of references to base game things.
E. g. Journal Entries and a few others.

With things like journal entries clicking on a working one that is defined in the same file as a non-working reference fixes the non-working reference.

This behaviour is really strange but I have not found any pattern yet.

What I tried to fix this:

- Invalidate all Indexes
- Uninstalling the Plugin and reinstalling the Plugin
- Restarting the IDE multiple times
